kategory/src/main/kotlin/kategory/instances/NumberInstances.kt

```diff
@@ -39,4 +39,4 @@ object FloatMonoid : Monoid<Float>, Semigroup<Float> by SGFloat, GlobalInstance<
     override fun empty(): Float = .0f
 }
 
-private val SGFloat: Semigroup<Float> = NumberSemigroup(Float::plus)
\ No newline at end of file
+private val SGFloat: Semigroup<Float> = NumberSemigroup(Float::plus)
```

Comment:
We should revisit providing these ambiguous instances until we have a proper way to have tagged types.
These instances are just for addition and questionably not precise some of the float ones in the JVM and as such we should either let the user provide them themselves or place them under `kategory.unsafe.*` or some other package. No need to do anything in this PR but thought I'd mentioning it since this `Semigroup<Float>` would not work for multiplication for example.